src/world/map.c

```diff
@@ -2235,7 +2243,7 @@ static int map_get_corner_height(rct_map_element *mapElement, int direction)
 static money32 smooth_land_tile(int direction, uint8 flags, int x, int y, int targetBaseZ, int minBaseZ)
 {
 	// Check if inside map bounds
-	if (x < 0 || y < 0 || x >= (256 * 32) || y >= (256 * 32)) {
+	if (!map_is_location_valid(x, y)) {
```

Comment:
Yes, I'm aware and I was just about to point this out.

What is `MAP_MINIMUM_X_Y`? I can do `<=` so the functionality will stay the same, but which one is correct?